app/views/static_pages/community_rules.html.erb

```diff
@@ -35,7 +35,7 @@
         <strong>Asking about or encouraging the use of Windows</strong>
       </li>
       <li>
-        <strong>Discussing mental health issues in depth or seeking mental health help from other users</strong>
+        <strong>Discussing unrelated topics that can be divisive or harmful</strong> such as politics, religion, relationships, mental health, medical concerns, homeopathic or medicinal remedies, etc.
```

Comment:
```suggestion
        <strong>Discussing unrelated topics that can be divisive or harmful</strong> such as politics, religion, relationships, mental health, medical concerns, homeopathic or other "alternative" remedies, etc.
```

Would this work?

Reply:
I prefer Rachel’s commit since “alternative” remedies sounds like “alternative medicine”. That would still not include regular prescription medicine.

Reply:
Ah, so that was the point of that wording. Wouldn't `medical concerns` cover that then? If not, possibly:
```suggestion
        <strong>Discussing unrelated topics that can be divisive or harmful</strong> such as politics, religion, relationships, mental health, medical concerns, prescription medicine, homeopathic or other "alternative" remedies, etc.
```

could make it crystal-clear?

Reply:
` <strong>Discussing unrelated topics that can be divisive or harmful</strong> such as politics, religion, relationships, mental health, medical concerns, homeopathic or medicinal remedies, etc.`

This is already an altered version done by Rachel based on my alteration to include medicine. I personally think `homeopathic or medicinal remedies` covers the load. 

Which problem are you trying to solve now, or asked differently; What do you believe is still missing? Is it the `or` part Rachel mentioned? I personally don’t think that looks out of place here. Plus your suggestion still includes the `or.

Reply:
Aight, I didn't really take "medicinal remedies" to mean legit prescription medicine due to its proximity to "homeopathic" and the use of "or".

`homeopathic or medicinal remedies` doesn't sound "right" to me. Might be an ESL kind of thing, might be the the proximity of those two words but "medicinal" sounds like "medicine-like" here and it could be either read as legit, prescription medicine (which is the intent) OR somewhat blurry reference to "alternative" medicine. I'd prefer them to become distinct parts of the sentence as proposed.

Reply:
> Wouldn't medical concerns cover that then? If not, possibly:

I don’t think it does. Asking about medical concerns is it of place. Asking about medicine is also out of place but is a different topic. Compare these two:

”I have had an itch on my left pinky for 3 days. Let me include some pictures, what do you all think this could be?”
”Hey, I have been taking prescription drugs xyz for a while, any other treatment you have tried and recommend?”

For me the difference would be in diagnosing medical concerns and asking for medical drug advice.

Reply:
Great conversation! I made some adjustments
1. use "medicine" instead  of "medicinal remedy", to be more clear
2. use "other home remedies" instead of "alternative", which I think covers the intent and does not need to be in quotes.
3. use "medical conditions" instead of "medical concerns" because the word stood out to me that it could cause confusion.
```suggestion
        <strong>Discussing unrelated topics that can be divisive or harmful</strong> such as politics, religion, relationships, mental health, medical conditions, medicine, homeopathic or other home remedies, etc.
```